2_process/src/process_and_style.R

```diff
@@ -1,16 +1,10 @@
-# Rename and remove unnecessary columns
-process_data <- function(nwis_data){
-  nwis_data_clean <- rename(nwis_data, water_temperature = X_00010_00000) %>% 
-    select(-agency_cd, -X_00010_00000_cd, -tz_cd)
-  
-  return(nwis_data_clean)
-}
-
-# Add site characteristic data
-annotate_data <- function(site_data_clean, site_filename){
+process_data <- function(nwis_data, site_filename){
   site_info <- read_csv(site_filename, show_col_types = FALSE)
-  annotated_data <- left_join(site_data_clean, site_info, by = "site_no") %>% 
+  
+  nwis_data_clean <- rename(nwis_data, water_temperature = X_00010_00000) %>% 
+    select(-agency_cd, -X_00010_00000_cd, -tz_cd) %>%
+    left_join(., site_info, by = "site_no") %>% 
```

Comment:
No changes needed, just a note for the future and for growing your `tidyverse` skill. Using `left_join()` after a `%>%` doesn't actually require that you specify the first input as `.` because `left_join()` will operate as the other dplyr functions do: the first argument by default is always the data frame being passed in from `%>%`. So, you _could_ write this simply is as `left_join(site_info, by = "site_no")` and it knows (and we, as the reader, know) that you are joining `site_info` _into_ the data frame being passed between the pipes (`%>%`).